x/gov/keeper.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/tendermint/tendermint/libs/log"
 )
 
+// TODO
```

Comment:
??

Reply:
it was for the linter, just updated to an actual description though